shell/browser/hid/electron_hid_delegate.cc

```diff
@@ -106,7 +107,9 @@ void ElectronHidDelegate::AddObserver(content::BrowserContext* browser_context,
 }
 
 void ElectronHidDelegate::RemoveObserver(
+    content::BrowserContext* browser_context,
     content::HidDelegate::Observer* observer) {
+  // DCHECK(base::Contains(observations_, browser_context));
```

Comment:
We should probably keep this DCHECK..?

Reply:
Oop, this is almost certainly a debugging change I left in, will uncomment